examples/portable-getusermedia/main.go

```diff
@@ -16,25 +18,25 @@ import (
 func main() {
 	var setting webrtc.SettingEngine
 
-	x264Params, _ := x264.NewParams()
 	openh264Params, _ := openh264.NewParams()
-	rtpTracker, _ := mediadevices.NewRTPTracker(x264Params, openh264Params)
-	rtpTracker.PopulateSetting(&setting)
```

Comment:
It should be automatically negotiated using `RTPSender.SetParameters()` and `RTPSender.Parameters()`.